CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,86 @@
+
+2. Be empathetic, welcoming, friendly, and patient. We work together to resolve conflict, and assume good intentions. We may all experience some frustration from time to time, but we do not allow frustration to turn into a personal attack. A community where people feel uncomfortable or threatened is not a productive one.
+
+3. Be collaborative. Our work will be used by other people, and in turn we will depend on the work of others. When we make something for the benefit of the project, we are willing to explain to others how it works, so that they can build on the work to make it even better. Any decision we make will affect users and colleagues, and we take those consequences seriously when making decisions.
+
+4. Be inquisitive. Nobody knows everything! Asking questions early avoids many problems later, so we encourage questions, although we may direct them to the appropriate forum. We will try hard to be responsive and helpful.
+
+5. Be careful in the words that we choose. We are careful and respectful in our communication and we take responsibility for our own speech. Be kind to others. Do not insult or put down other participants. We will not accept harassment or other exclusionary behaviour, such as:
+
+
+> * Violent threats or language directed against another person.
+> * Sexist, racist, ableist, or otherwise discriminatory jokes and language.
+> * Posting sexually explicit or violent material.
+> * Posting (or threatening to post) other people’s personally identifying information (“doxing”).
+> * Sharing private content, such as emails sent privately or non-publicly, or unlogged forums such as IRC channel history, without the sender’s consent.
+> * Personal insults, especially those using sexist, racist, or ableist terms.
+> * Intentional or repeated misgendering of participants who have explicitly requested to be addressed by specific pronouns.
+> * Unwelcome sexual attention.
+> * Excessive profanity. Please avoid swearwords; people differ greatly in their sensitivity to swearing.
+> * Repeated harassment of others. In general, if someone asks you to stop, then stop.
+> * Advocating for, or encouraging, any of the above behaviour.
+
+## Diversity Statement
+The napari project welcomes and encourages participation by everyone. We are committed to being a community that everyone enjoys being part of. Although we may not always be able to accommodate each individual’s preferences, we try our best to treat everyone kindly.
+
+No matter how you identify yourself or how others perceive you: we welcome you. Though no list can hope to be comprehensive, we explicitly honour diversity in: age, culture, ethnicity, genotype, gender identity or expression, language, national origin, neurotype, phenotype, political beliefs, profession, race, religion, sexual orientation, socioeconomic status, subculture and technical ability, to the extent that these do not conflict with this code of conduct.
```

Comment:
"to the extent that these do not conflict with this code of conduct" Oooh, nice way to dodge the [paradox of tolerance](https://en.wikipedia.org/wiki/Paradox_of_tolerance). =D